cmd/fs-v1.go

```diff
@@ -695,13 +694,12 @@ func (fs *FSObjects) defaultFsJSON(object string) fsMetaV1 {
 	fsMeta := newFSMetaV1()
 	fsMeta.Meta = make(map[string]string)
 	fsMeta.Meta["etag"] = defaultEtag
-	if objectExt := path.Ext(object); objectExt != "" {
-		if content, ok := mimedb.DB[strings.ToLower(strings.TrimPrefix(objectExt, "."))]; ok {
-			fsMeta.Meta["content-type"] = content.ContentType
-		} else {
-			fsMeta.Meta["content-type"] = "application/octet-stream"
-		}
+	// If not set default to "application/octet-stream"
+	contentType := mimedb.TypeByExtension(path.Ext(object))
```

Comment:
we should return to `application/octet-stream`

Reply:
Done